sorc/fvcom_tools.fd/module_nwp.f90

```diff
@@ -23,44 +26,53 @@ module module_nwp
 
    private
    type :: nwp_type
-      character(len=6) :: datatype
-      integer :: numvar, xlat, xlon, xtime
-      integer :: i_mask, i_sst, i_ice, i_sfcT, i_iceT
-      character(len=20), allocatable :: varnames(:)
-      character(len=20), allocatable :: latname
-      character(len=20), allocatable :: lonname
-      character(len=20), allocatable :: dimnameEW
-      character(len=20), allocatable :: dimnameNS
-      character(len=20), allocatable :: dimnameTIME
-
-      real(r_kind), allocatable :: nwp_mask(:,:,:)
-      real(r_kind), allocatable :: nwp_sst(:,:,:)
-      real(r_kind), allocatable :: nwp_ice(:,:,:)
-      real(r_kind), allocatable :: nwp_sfcT(:,:,:)
-      real(r_kind), allocatable :: nwp_iceT(:,:,:) 
+      character(len=6) :: datatype !< Data type.
+      integer :: numvar !< Number of variabls.
+      integer :: xlat !< Number of latitudes.
+      integer :: xlon !< Number of longitudes.
+      integer :: xtime !< Number of times.
+      integer :: i_mask !< Is var visible (always 1).
+      integer :: i_sst !< Index of sst var.
+      integer :: i_ice !< Index of ice var.
+      integer :: i_sfcT !< Index of sst temp var.
+      integer :: i_iceT !< Index of ice temp var.
+      character(len=20), allocatable :: varnames(:) !< Variable names.
+      character(len=20), allocatable :: latname !< Latitude name.
+      character(len=20), allocatable :: lonname !< Longitude name.
+      character(len=20), allocatable :: dimnameEW !< East/West dimension name.
+      character(len=20), allocatable :: dimnameNS !< North/South dimension name.
+      character(len=20), allocatable :: dimnameTIME !< Time dimension name.
+
+      real(r_kind), allocatable :: nwp_mask(:,:,:) !< ???
```

Comment:
@dmwright526 can you tell me what these variables are? (The ones documented with '???')

Reply:
@edwardhartnett Just updated with a description for these variables.